script.js

```diff
@@ -1,4 +1,65 @@
+//input = Player1, roll 2 dice, order of guess (dice1 or dice2)
+//output = Player2, output 2 guesses ... username2, output 2 guesses + outcome (who wins/lose)
+var randomWord = "Keyboard match";
+var gameMode = "enter username";
+var username = "";
+var myOutputValue = "";
+var Player1 = 0;
+var Player2 = 0;
+var numberOfTries = 0;
+
+var rollDice = function () {
+  var randomNumber = Math.floor(Math.random() * 7);
+  return randomNumber;
+};
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "hello world";
+  var myOutputValue = "Try Again";
+  if (gameMode == "enter username") {
+    console.log("username entered");
+    username = input;
+    gameMode = "game start";
+    myOutputValue = `Welcome ${username} <br><br> Press Submit to roll Dice. `;
+    console.log("dice game starts");
+    return myOutputValue;
+  }
+  if (gameMode == "game start") {
```

Comment:
For all the modes (e.g. "game start") in this case, consider storing such string constants as variables. This helps you avoid spelling errors if you need to re-use these modes at some point of the code